cmd/get.go

```diff
@@ -50,11 +51,21 @@ and provides a fast and easy alternative to a package manager.`,
 	command.Flags().StringP("output", "o", "", "Output format of the list of tools (table/markdown/list)")
 	command.Flags().Bool("stash", true, "When set to true, stash binary in HOME/.arkade/bin/, otherwise store in /tmp/")
 	command.Flags().StringP("version", "v", "", "Download a specific version")
+	command.Flags().Bool("dry-run", false, "When set to true, performs a dry run of binary download and install location")
 
 	command.RunE = func(cmd *cobra.Command, args []string) error {
+		dry_run, _ := command.Flags().GetBool("dry-run")
+
 		if len(args) == 0 {
 			output, _ := command.Flags().GetString("output")
 
+			if dry_run {
+				fmt.Println(`Cannot use '--dry-run' flag without tool name.
+Get help using:
```

Comment:
```suggestion
Find help on the `get` command by using:
```
This is subtle and nitty but I think it will help newer users understand the nature of `--help`.   Further explanation would be from the opposite angle.... an unfamiliar user might misunderstand that the Arkade help command is more idiomatic `arkade 'get help'`, which could cause them to miss the rest of the help scope.